javascript/javascript1/week1/smart-ease.js

```diff
@@ -0,0 +1,54 @@
+// Age-ify (A future age calculator)
+
+// year of my birth
+const yearOfBirth = 2000;
+
+// future year
+const  yearFuture = 2058;
+
+// how old will i be
+const age = yearFuture - yearOfBirth;
+console.log("I will be "+ age + " years old in " + yearFuture + "." )
+
+
+//Goodboy-Oldboy (A dog age calculator)
+// year of dog's birth
+const dogYearOfBirth = 2017;
+
+// future year for dog
+const  dogYearFuture = 2027;
+
+// how old will the dog be
+const dogYear = dogYearFuture - dogYearOfBirth;
+
+const shouldShowResultInDogYears = true
+
+if (shouldShowResultInDogYears){
+    console.log("Your dog will be "+ dogYear*7 + " dog years old in " + dogYearFuture + "." )
+}else{
+    console.log("Your dog will be "+ dogYear + " human years old in " + dogYearFuture + "." )
+}
```

Comment:
Overall good work. It's great to see you close your statements with ";" continuously. You could add ";" to the end og line 27 and 29 as well. 

Your logic is good. But a little refactoring can be made 💡
I suggest you calculate the value of dogYear in line 22 like const dogYear = (dogYearFuture - dogYearOfBirth)*7
Additionally, you can add a variable that stores the value of humanYear, which you can apply in line 29. 

If you want to upgrade the code even more, I suggest you look into template literals, which will let you work with strings in an even more elegant way 😊
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals